.github/workflows/ci.yml

```
@@ -48,6 +48,7 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
```

```suggestion
          - "3.11"
```

<-- autoblocks-trace-id=2343243432 -->